deep_quoridor/src/quoridor_env.py

```diff
@@ -521,7 +559,10 @@ def distance_to_target(self, row, col, target_row, any_path=False):
         Otherwise, the shortest path will be returned (potentially slower)
         """
         visited = np.zeros((self.board_size, self.board_size), dtype="bool")
-        return self._dfs(row, col, target_row, visited, any_path)
+        if any_path:
```

Comment:
I think the API for this is getting confusing.  I'd keep `can_reach` that can directly call _dfs, and in this method remove `any_path` and call _dfs, so that we can get the minimum distance (I feel like when you say distance it's usually the minimum)